lib/rust/parser/generate-java/java/org/enso/syntax2/Parser.java

```diff
@@ -8,6 +8,12 @@
 
 public final class Parser implements AutoCloseable {
   private static void initializeLibraries() {
+    try {
+      System.loadLibrary("enso_parser");
```

Comment:
@kazcw, your review is needed (probably) because of this `Parser` change. Rather than trying to locate the library by `searchFromDirToTop` it, first of all just asks the system to load the library itself. The _NetBeans Runtime Container_ [specifies how to package the libraries](https://web.archive.org/web/20210118080511/https://platform.netbeans.org/articles/installation.html) for different architectures:

> lib: contains native JNI libraries and basic boot files. The naming policy of the libraries is in hands of the modules, as that it has worked well enough till now. If it proves unsufficient following conventions shall be obeyed: There are subdirectories under the lib directory that reflect the architecture (e.g. i386, obtained via System.getProperty ("os.arch")) and possibly also subdirectory identifying the system (e.g. i386/linux, obtained via System.getProperty ("os.name").toLowerCase ()). When the system is looking for a particular library it scans the deepest directories first and then the shallow ones (lib/i386/linux, lib/i386, lib). 

Currently the [enso4igv-1.36.106.nbm](https://github.com/enso-org/enso/actions/runs/9500540370) file contains:
```
$ unzip -l enso4igv-1.36.106.nbm | grep lib/ | cut -c31-
netbeans/modules/lib/
netbeans/modules/lib/aarch64/
netbeans/modules/lib/aarch64/libenso_parser.dylib
netbeans/modules/lib/enso_parser.dll
netbeans/modules/lib/libenso_parser.dylib
netbeans/modules/lib/libenso_parser.so
```
With some luck the `System.loadLibrary("enso_parser")` will load the right library on each of the supported systems.